add-ons/fabric8/README.md

```diff
@@ -0,0 +1,50 @@
+# Fabric8 Addon
+
+This addon provides an easy way to install fabric8 on MiniShift. 
+
+Fabric8 provides a complete developer platform for creating projects, editing them and performing full CI / CD for the entire developer lifecycle.
+
+## Setup GitHub OAuth client ID and secret
+
+Fabric8 has GitHub integration letting you browse repositories, create new repositories, edit projects and setup automated CI / CD jobs with webhooks on github. 
+
+This requires an [OAuth application to be setup on your github account](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps/) for fabric8 and you need to obtain the client ID and secret for the OAuth application.
```

Comment:
@jstrachan, let me refine this. I think it is working. I can log into the OpenShift console and all pods are running. I also can open this route  http://fabric8-fabric8.192.168.99.203.nip.io, but then I am a bit lost. How do I log in? What credentials are expected here? Maybe we can add something to the README and/or add an additional echo to the add-on with a link to where to go once Fabric8 is up. WDYT?